assets/css/main.css

```
@@ -170,6 +170,28 @@ body {
   text-decoration: none;
 }
 
+#Explanation{
+  font-size: 12pt; 
+}
+
+#Toggle-Explanation{
+  background: #f1652991;
+  font-size: 12pt;
```

You can use `rem` instead of `pt`

```suggestion
  font-size: 1rem; 
```